src/AutorestSwift/View Models/RequestViewModel.swift

```diff
@@ -47,10 +47,10 @@ struct RequestViewModel {
     init(from request: Request, with operation: Operation) {
         // load HttpRequest properties
         let httpRequest = request.protocol.http as? HttpRequest
-        let uri = httpRequest?.uri ?? nil
+        self.uri = httpRequest?.uri ?? nil
```

Comment:
There's no reason to have `?? nil` since the point of `??` is to use the rhs when the lhs is nil